locations/spiders/uniteddairyfarmers.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+import re
+import json
+
+import scrapy
+
+from locations.items import GeojsonPointItem
+from locations.hours import OpeningHours
+
+class UnitedDairyFarmersSpider(scrapy.Spider):
+    download_delay = 0.2
+    name = "UnitedDairyFarmers"
```

Comment:
We usually use `snake_case` for spider names. Can you change this one please?

```suggestion
    name = "united_dairy_farmers"
```